modules/va_forms/app/controllers/va_forms/v0/forms_controller.rb

```diff
@@ -17,6 +16,16 @@ def show
         render json: forms,
                serializer: VaForms::FormDetailSerializer
       end
+
+      private
+
+      def get_forms
+        if params[:query].present?
+          Form.where('name like ? OR title like ?', "%#{params[:query]}%", "%#{params[:query]}%")
```

Comment:
Can you use % instead of like for the search operator, might need to include pg_trim but will give you way more fuzzy searches in a way then just a like term https://www.rdegges.com/2013/easy-fuzzy-text-searching-with-postgresql/

Reply:
Also this should be form_name over name

Reply:
just using ilike, the other seems to be a little heavy handed for this initial release. will keep it in our back pockets for future requests